src/json_spirit/CMakeLists.txt

```diff
@@ -2,3 +2,6 @@ add_library(json_spirit STATIC
   json_spirit_reader.cpp
   json_spirit_writer.cpp)
 target_link_libraries(json_spirit common_utf8)
+if(WIN32)
+  target_link_libraries(json_spirit Boost::thread)
+endif()
```

Comment:
i'd avoid fixes like this. if json_spirit depends on Boost::thread, we should link it against Boost::thread on all platforms.

Reply:
Right, there's no reason to add this platform check.